src/Protocol/Client/Capabilities/TextDocumentClientCapabilities.cs

```diff
@@ -109,7 +109,7 @@ public class TextDocumentClientCapabilities
         ///
         /// Since 3.10.0
         /// </summary>
-        public Supports<FoldingRangeCapability> FoldingRangeProvider { get; set; }
+        public Supports<FoldingRangeCapability> FoldingRange { get; set; }
```

Comment:
Please note that this is a breaking change. But as nobody complained that the capability is not working I assume very little people are using it :)

Reply:
Nice catch! Yes, this breaking change is no problem as it didn’t even work in the first place 😄